src/buildInfo.json

```diff
@@ -0,0 +1 @@
+{"timestamp":"","gitHashShort":"","gitBranch":""}
```

Comment:
This should not be present in the PR. please remove. We can ignore that tslint error about missing file.